fmriprep/interfaces/reports.py

```diff
@@ -194,10 +194,12 @@ def _generate_segment(self):
                 '(boundary-based registration, BBR) - %d dof' % dof,
                 'FreeSurfer <code>mri_coreg</code> - %d dof' % dof],
         }[self.inputs.registration][self.inputs.fallback]
-        if self.inputs.pe_direction is None:
-            pedir = 'MISSING - Assuming Anterior-Posterior'
-        else:
-            pedir = {'i': 'Left-Right', 'j': 'Anterior-Posterior'}[self.inputs.pe_direction[0]]
+        pedir = {
+            'i-': 'Left-Right',
+            'i': 'Right-Left',
+            'j-': 'Anterior-Posterior',
+            'j': 'Posterior-Anterior'
+        }.get(self.inputs.pe_direction, 'MISSING - Assuming Anterior-Posterior')
```

Comment:
This is only true for LAS images. We need the orientation of the original image, to be certain.

Reply:
Good point - I've added a commit to consider LAS/RAS. How extensive do you think our support should be for different orientations?